README.md

```diff
@@ -64,6 +64,18 @@ We support the following CMake options:
 
 I highly recommend the ccache options. They don't speed up the first clean build, but they speed up future clean builds after the first build.
 
+### Installing for use with mandel.cdt (or others)
+
+```
+git submodule update --init --recursive
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=/usr/local ..
```

Comment:
We don't want to encourage users to install dev components in to /usr/local because it causes conflicts. Will need to use something else (idk what we've been indecisive; maybe $HOME/mandeldev :shrug: ).

Reply:
Update: 2d05d35